contributed_definitions/NXem.nxdl.xml

```diff
@@ -358,6 +753,31 @@ the fields they can be used-->
                                 <field name="voltage" type="NX_NUMBER"/>
                             </group>
                         </group>
+                        <group type="NXscanbox_em" optional="true">
+                            <field name="scan_schema" type="NX_CHAR"/>
+                            <field name="dwell_time" type="NX_NUMBER"/>
+                        </group>
+                        <group type="NXstage_lab" optional="true"/>
+                        <group type="NXdetector" optional="true">
+                            <field name="mode" type="NX_CHAR"/>
+                        </group>
+                        <group name="heater" type="NXobject" optional="true">
```

Comment:
Does not it suggest to make an NXheater base class? ON the other hand, should not such also contain NXfabriation, too?

Reply:
I had such a base class it was argued against in that it should be an NXactuator instead which is why I removed it, I feel we are running in circles. NXheater is essentially a furnace a furnace is not only its heating element that is indeed an actuator. So having had NXheater made sense to me.